FhirLoader.Tests.E2E/UnitTest.cs

```diff
@@ -0,0 +1,83 @@
+using FhirLoader.Tool;
+
+namespace FhirLoader.Tests.E2E
+{
+    public class UnitTest
+    {
+        private const string fhirURL = "https://workpacesdkfhir-sjbuildfhir.fhir.azurehealthcareapis.com/";
+        private const int batchsize = 500;
+        private const int concurrency = 50;
+
+        /// <summary>
+        /// E2E Test Case for bundle Files.
+        /// By passing FolderPath,it will read all the files and processed to the Fhir URL. 
+        /// </summary>
+        [Fact]
+        public async void LocalFhir_bundle_Test()
+        {
+            CommandOptions commandOptions = new()
+            {
+                FhirUrl = fhirURL,
+                FolderPath = @"E:\Projects\fhir-loader\TestSamples\bundle",
+                BatchSize = batchsize,//Must be betweeen 1 & 500,Default 500
+                Concurrency = concurrency//Must be betweeen 1 & 50,Default 50
+            };
+            int response = await Program.Run(commandOptions);
+            Assert.Equal(1, response);
```

Comment:
I would expect the test to query the FHIR Service after the test and ensure that the resources were actually loaded. E2E tests don't need to test the output of the program to determine success or failure - they can test the behavior the program has on external systems.